sql/mssql/00003-Modify-Users-Add-SoftDelete.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE [eppeta].[Users] ADD [DeletedAt] DATETIME NULL;
+GO
```

Comment:
This allows for a "soft delete", where the record is never actually deleted but just "marked as deleted".